.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+      - run: cargo run --example file_parser examples/sdps/08.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/09.sdp
+      - run: cargo run --example file_parser examples/sdps/10.sdp
+      - run: cargo run --example file_parser examples/sdps/11.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/12.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/13.sdp
+      - run: cargo run --example file_parser examples/sdps/14.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/15.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/16.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/17.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/18.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/19.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/20.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/21.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/22.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/23.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/24.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/25.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/26.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/27.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/28.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/29.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/30.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/31.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/32.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/33.sdp --expect-failure
+      - run: cargo run --example file_parser examples/sdps/34.sdp
+      - run: cargo run --example file_parser examples/sdps/35.sdp
+      - run: cargo run --example file_parser examples/sdps/36.sdp
+      - run: cargo run --example file_parser examples/sdps/37.sdp
+      - run: cargo run --example file_parser examples/sdps/38.sdp
+      - run: cargo run --example file_parser examples/sdps/39.sdp
+      - run: cargo run --example file_parser examples/sdps/40.sdp
+      - run: cargo run --example file_parser examples/sdps/41.sdp --expect-failure
+      #TODO run code coverage
+
+#TODO split into build and test
+
+workflows:
+  workflow:
+    jobs:
+      - build:
+          matrix:
+            parameters:
+              features: ["", "serialize"]
+              rustversion: ["1.45.0", "1.46.0", "1.47.0", "1.48.0", "1.49.0"]
```

Comment:
Lets change this to min and max version. I think running with all the versions in between is not very likely to find new issues.